scramble/scramble-text.sql

```diff
@@ -58,7 +58,19 @@ select @rowcount=@@ROWCOUNT, @error=@@ERROR
 if @error<>0 goto ERR
 print @rowcount + ' strings scrambled'
 
-update dbo.LongString set Value=dbo.__RandomString('This Is a Random Description ' + cast(NEWID() as varchar(100)))
+if exists (select * from INFORMATION_SCHEMA.TABLES where TABLE_SCHEMA='dbo' and TABLE_NAME='BaseAssetTaggedWith' and TABLE_TYPE='BASE TABLE') begin
+	create table #tags (Tag nvarchar(440) collate Latin1_General_CI_AS not null primary key, Result nvarchar(440) collate Latin1_General_CI_AS not null unique)
```

Comment:
since `Result` has a unique constraint, I guess we'll just let it fail if it happens to duplicate a scrambled tag.  we can always tell them to try again

Reply:
yup. that was my thinking exactly